grails-app/domain/org/pih/warehouse/inventory/CycleCountCandidate.groovy

```diff
@@ -30,15 +32,15 @@ class CycleCountCandidate {
 
     Integer negativeItemCount
 
-    Date dateLastCount
+    Instant dateLastCount
 
-    Date dateNextCount
+    Instant dateNextCount
 
     Integer daysUntilNextCount
 
     Boolean hasStockOnHandOrNegativeStock
 
-    Date dateLatestInventory
+    Instant dateLatestInventory
```

Comment:
- dateLastCount: based on CycleCountItem.dateCounted and/or Transaction.transactionDate which are both date-time fields (transactionDate is a Date in the domain but since we're populating the view via sql query we don't interact with the Transaction domain and so can use Instant)
- dateNextCount: based on dateLastCount (takes the date and adds X days to it)
- dateLatestInventory: unused and always null. I picked Instant because that's what made sense to me